test/ng/directive/ngOptionsSpec.js

```diff
@@ -1138,13 +1138,13 @@ describe('ngOptions', function() {
       });
 
       scope.$apply(function() {
-        scope.selected = scope.arr[0];
+        scope.selected = {id: 10, label: 'ten'};
       });
 
       spyOn(element.controller('ngModel'), '$render');
 
       scope.$apply(function() {
-        scope.selected.label = 'changed';
+        scope.arr[0].id = 20;
```

Comment:
Because we are using `track by` here, the model and the option are different objects.

So we are not interested in the `label` of the selected item, as this doesn't affect the visual display of the options, which are only interested in the `label` property on the options themselves.